atacac/differentiate.py

```diff
@@ -1,43 +1,43 @@
-import os
 import json
 import glob
 
-import yaml
 import dictdiffer
 import click
 from tower_cli.exceptions import TowerCLIError
+import yaml
 
-from atacac._utils import log, tower_receive
+from atacac._utils import log, tower_receive, load_asset
 
 
 @click.command()
 @click.argument('assets_glob', envvar='ASSETS_GLOB')
 def main(assets_glob):
     diff = False
 
-    for file_name in glob.glob(assets_glob, recursive=True):
+    for file_name in sorted(glob.glob(assets_glob, recursive=True)):
+        asset = load_asset(file_name)
+
         try:
-            jt_name = os.path.splitext(file_name)[0].replace('_', ' ')
-            jt_data = tower_receive('job_template', jt_name)[0]
+            jt_data = tower_receive(asset['asset_type'], asset['name'])[0]
         except TowerCLIError:
-            log('INFO', (f"Asset '{jt_name}' doesn't exist in Tower, no need "
-                         "to check for diffs"))
+            log('INFO', (f"Asset '{asset['name']}' doesn't exist in Tower, no "
+                         "need to check for diffs"))
             continue
 
-        jt_file_src = file_name
-        jt_file_src_data = yaml.load(open(jt_file_src), Loader=yaml.FullLoader)
-
-        log('INFO', f"Differentiating '{jt_file_src}' and '{jt_name}'")
+        # Need to parse extra vars to dict becuse in assets file it is YAML and
```

Comment:
We should note this in docs/README.md#notes too

Reply:
I believe this is just implementation detail, user don't need to know about it.